frappe/public/js/frappe/chat.js

```diff
@@ -177,6 +177,60 @@ frappe.datetime.compare = (a, b) => {
 		return  0
 }
 
+// frappe.quick_edit
+frappe.quick_edit      = (doctype, docname, fn) => {
+	return new Promise(resolve => {
+		frappe.model.with_doctype(doctype, () => {
+			frappe.db.get_doc(doctype, docname).then(doc  => {
```

Comment:
either use `model.with_doc` or `db.get_doc` not `model.with_doctype`

Reply:
Ok.

Reply:
@netchampfaris I don't think `model.with_doc` or `db.get_doc` doesn't cache `meta` description.